src/character.cpp

```diff
@@ -454,7 +454,8 @@ Character::Character() :
     update_type_of_scent( true );
     pkill = 0;
     // 45 days to starve to death
-    healthy_calories = 55000;
+    // 55 Mcal or 55k kcal
+    healthy_calories = 55'000'000;
```

Comment:
Sorry to stick my uneducated nose in here, but is `55'000'000` a valid assignment as opposed to just `55000000`?

Reply:
Yes, they're called digit separators and were introduced in C++14.